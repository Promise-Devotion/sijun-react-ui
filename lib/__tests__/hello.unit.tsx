// describe(Button.displayName, () => {
//     it('DisplayName', () => {
//       expect(Button.displayName).toEqual('Button')
//     })

import renderer from 'react-test-renderer'
import React from 'react'
import Button from '../button'

describe('button', () => {
    it('是一个div', ()=> {
        const json = renderer.create(<Button />).toJSON()
        expect(json).toMatchSnapshot()
    })
})