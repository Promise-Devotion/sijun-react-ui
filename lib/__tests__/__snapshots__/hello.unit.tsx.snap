// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 是一个div 1`] = `
<div>
  按钮
</div>
`;
